src/strongbox/jfx.clj

```diff
@@ -2262,21 +2274,71 @@
         blacklist [:group-addons :release-list :source-map-list]
         sanitised (apply dissoc addon blacklist)
 
+        transformations {:interface-version str}
+        sanitised (apply (fn [[key valfn]]
+                           (update addon key valfn)) transformations)
+
         row-list (apply utils/csv-map [:key :val] (vec sanitised))
-        row-list (sort-by :key row-list)]
-    {:fx/type :border-pane
-     :top {:fx/type :label
-           :style-class ["section-title"]
-           :text "raw data"}
-     :center {:fx/type :table-view
-              :id "key-vals"
-              :style-class ["table-view"]
-              :placeholder {:fx/type :text
-                            :style-class ["table-placeholder-text"]
-                            :text "(not installed)"}
-              :column-resize-policy javafx.scene.control.TableView/CONSTRAINED_RESIZE_POLICY
-              :columns (mapv make-table-column column-list)
-              :items (or row-list [])}}))
+        row-list (sort-by :key row-list)
+
+        addon-string (if (:version addon)
+                       (format "%s\n\nVersion %s\n\n\"%s\"\n\nInstalled from %s\n\nSupports %s\n\nLast updated %s (%s)"
+                               (:label addon)
+                               (:version addon)
+                               (:description addon)
+                               (:source addon)
+                               (clojure.string/join ", " (mapv sp/game-track-labels-map (:supported-game-tracks addon)))
+                               (-> addon :updated-date utils/format-dt)
+                               (-> addon :updated-date utils/datestamp-ymd))
+
+                       ;; catalogue item
+                       (format "%s\n\n\"%s\"\n\nAvailable from %s\n\nSupports %s\n\nLast updated %s (%s)"
+                               (:label addon)
+                               (:description addon)
+                               (:source addon)
+                               (clojure.string/join ", " (mapv sp/game-track-labels-map (:game-track-list addon)))
+                               (-> addon :updated-date utils/format-dt)
+                               (-> addon :updated-date utils/datestamp-ymd)))
```

Comment:
stick these two into separate functions and test

and shouldn't they be using 'sanitised' rather than 'addon' ?